app/src/main/java/com/github/HumanLearning2021/HumanLearningApp/view/GoogleSignInFragment.kt

```diff
@@ -11,11 +11,8 @@ import android.widget.Button
 import android.widget.TextView
```

Comment:
Should this fragment follow its test?

Reply:
I'm not really sure what you mean by this, but do you have a suggestion for where to put this fragment? I wasn't sure where it would fit best

Reply:
I was thinking most things in the `fragments` package are really widgets, especially given that activities are now going to be fragments so I would call them that and run some renames...

Reply:
Oh and my original comment was about this class not being in the same package as its `-Test' but that's fixed

Reply:
Is it ok if I let you do the renaming? I'm not really sure how you would do the renaming. I already deleted the fragments package because it didn't make sense to keep it since we will turn everything into fragments. I think it's best if we make packages by logical units (like I did with the dataset_list_fragment)

Reply:
It's just doing smart renames in Studio but I'll do it now

Reply:
here you go!